examples/complete/main.tf

```diff
@@ -350,11 +350,11 @@ module "ec2_capacity_reservation" {
   instance_type               = "t3.micro"
   subnet_id                   = element(module.vpc.private_subnets, 0)
   vpc_security_group_ids      = [module.security_group.security_group_id]
-  associate_public_ip_address = true
+  associate_public_ip_address = false
 
   capacity_reservation_specification = {
     capacity_reservation_target = {
-      capacity_reservation_id = aws_ec2_capacity_reservation.targeted.id
```

Comment:
I think we want one targeted example https://github.com/terraform-aws-modules/terraform-aws-autoscaling/blob/master/examples/complete/main.tf#L468-L472

and one open example https://github.com/terraform-aws-modules/terraform-aws-autoscaling/blob/master/examples/complete/main.tf#L222-L224

so we'll need to add an open example